bee-storage/bee-storage/src/lib.rs

```diff
@@ -20,3 +20,4 @@
 pub mod access;
 pub mod backend;
 pub mod health;
```

Comment:
Maybe then put health in system together with version ?

Reply:
makes sense